test/Microsoft.DocAsCode.MarkdigEngine.Tests/XrefTest.cs

```diff
@@ -17,6 +17,7 @@ public void XrefTestGeneral()
 @Microsoft.Build.Tasks
 ""@""a[test](link)
 @hehe
+![img @hehe a](src ""title @hehe b"")
```

Comment:
Would you also add cases like "![img @hehe a" and `![img @hehe a](src ""title @hehe b""`

Reply:
Also `![this **img @hehe a** haha](src ""title @hehe b"")`

Reply:
@yufeih From the test case, For "![img @hehe a", the XrefInlineShortParser will not work also. Is there any way to fix this case?

Reply:
Fixed in Renderer